client/reader/discover/follow-button.jsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import FollowButton from 'reader/follow-button';
+import * as discoverStats from './stats';
+
+class DiscoverFollowButton extends React.Component {
```

Comment:
Good point! `localize()` will accept a function too, so this could potentially be refactored to a stateless functional component. I will save for a future PR though, if I may :)